R-package/R/lightgbm.R

```diff
@@ -91,7 +91,8 @@ NULL
 #' @description Simple interface for training a LightGBM model.
 #' @inheritParams lgb_shared_params
 #' @param label Vector of labels, used if \code{data} is not an \code{\link{lgb.Dataset}}
-#' @param weight vector of response values. If not NULL, will set to dataset
+#' @param weights Sample / observation weights for rows in the input data. If `NULL`, will assume that all
```

Comment:
```suggestion
#' @param weights Sample / observation weights for rows in the input data. If \code{NULL}, will assume that all
```

Please use `\code{}` instead of backticks. This project's roxygen docs are currently written using LaTeX, not markdown.

Reply:
even if it does, please use `\code{}` for consistency with the rest of the package's code.

Reply:
Seems they are actually different formats in R's documentation files, both of them written verbatim in the .Rd, although they look the same in my setup.

Reply:
Great, thank you for updating them in https://github.com/microsoft/LightGBM/pull/4975/commits/314e6979b49b704e450761a5d7be264228f32eec